handsontable/src/i18n/languages/ar.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * @preserve
+ * Authors: Handsoncode
+ * Last updated: Feb 9, 2022
+ *
+ * Description: Definition file for Arabic - Without a specific country.
+ */
+import * as C from '../constants';
+
+const dictionary = {
+  languageCode: 'ar',
```

Comment:
```suggestion
  languageCode: 'ar-AR',
```
In reference to the best practices for language tags ([BCP 47](https://www.rfc-editor.org/rfc/rfc5646.html)), may we agree to use the `ar-AR` tag here?